antarest/study/storage/rawstudy/io/reader/ini_reader.py

```diff
@@ -77,6 +77,53 @@ def read(self, path: Path) -> JSON:
         }
 
 
+class XpansionSettingsIniReader(IReader):
+    """
+    Standard .ini file reader. Use for general purpose.
+    """
+
+    @staticmethod
+    def _parse_inf(value: str) -> Optional[str]:
+        try:
+            return "inf" if float(value) == float("inf") else None
+        except ValueError:
+            return None
+
+    @staticmethod
+    def parse_value(value: str) -> ELEMENT:
+        parsed: Union[
+            str, int, float, bool, None
+        ] = XpansionSettingsIniReader._parse_inf(value)
+        parsed = parsed if parsed is not None else IniReader._parse_bool(value)
+        parsed = parsed if parsed is not None else IniReader._parse_int(value)
+        parsed = (
+            parsed if parsed is not None else IniReader._parse_float(value)
+        )
+        return parsed if parsed is not None else value
+
+    @staticmethod
+    def _parse_json(json: configparser.SectionProxy) -> JSON:
+        return {
+            key: XpansionSettingsIniReader.parse_value(value)
+            for key, value in json.items()
+        }
+
+    def read(self, path: Path) -> JSON:
+        config = IniConfigParser()
+
+        with open(path, "r") as f:
+            json = {}
+            for line in f.readlines():
+                line = line.strip()
+                try:
+                    json[line.split("=")[0]] = line.split("=")[1]
+                except IndexError:
+                    pass
```

Comment:
just split the string once, don't catch the IndexError exception since we want the exception to be raised if the file is not ok, strip the key and value from the previous split